src/Definition/Source/SourceCache.php

```diff
@@ -23,21 +23,39 @@ class SourceCache implements DefinitionSource, MutableDefinitionSource
      */
     private $cachedSource;
 
+	/**
+	 * @var null|string
+	 */
+	private static $cacheKeyPrefix;
```

Comment:
Is it necessary to use a static property?

Reply:
yes